### DISCUSSION TITLE
✅ 🚀 Feature Request: Loop Accordion Widget

### DISCUSSION THREAD

**Discussion by HQRaja** [elementor/elementor#23240] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Allow building an accordion with content from a query.

Use case: FAQs as CPT could be dynamically queried and displayed in accordion format without having to statically add them to an accordion widget in each part of the site where they are to be displayed. This could be combined with supported third-party facet/filter/search plugins to filter them out like any other query loop.

### Describe the solution you'd like

Simply replicate the existing accordion widget with the ability to specify a query as the data source for the accordion items rather than requiring the content to be entered in the widget settings itself. (The way the Loop Carousel widget is to the Carousel widget, pretty much!)

**Comment by nicholaszein** [comment]
Hi @HQRaja!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with the new Nested Accordion widget in Elementor v3.15.0! 🥳 

You can now add this widget inside a Loop Item template, set the title to your post-title, add the Post Content widget inside, and use it as a Loop Accordion. You can even decide if the Accordion will be open or closed by default. 😉 💪 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
